.pylintrc

```diff
@@ -45,7 +46,7 @@ limit-inference-results=100
 
 # List of plugins (as comma separated values of python module names) to load,
 # usually to register additional checkers.
-load-plugins=pylint_pytest
+load-plugins=
```

Comment:
Yes, I saw this was causing problems. I can't remember what it was needed for. I think pylint gets confused by fixtures without it.

Reply:
Ah, just saw your explanation in the intro. That makes sense now.